install-dev/data/db_structure.sql

```diff
@@ -47,7 +47,7 @@ CREATE TABLE `PREFIX_address` (
 /* Used for search, if a search string is present inside the table, search the alias as well */
 CREATE TABLE `PREFIX_alias` (
   `id_alias` int(10) unsigned NOT NULL auto_increment,
-  `alias` varchar(191) NOT NULL,
+  `alias` varchar(255) NOT NULL,
```

Comment:
This should not be changed and rather fixed on the ObjectModel level, see: https://github.com/PrestaShop/PrestaShop/commit/61b47cce5b701bf92d091339c359e49ce3b2fe10

This is only for those fields which are indexes.

This PR should go to the develop branch to make sure we won't mistakenly introduce problems in a patch version.

Reply:
Indeed you should set 191 as the limit in Address::alias instead

Reply:
Resolved